jplag/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java

```diff
@@ -154,12 +154,12 @@ private static List<FilesOfSubmission> getFilesForSubmission(Submission submissi
      * @return A Match DTO.
      */
     private static Match convertMatchToReportMatch(JPlagComparison comparison, de.jplag.Match match, Boolean usesIndex) {
-        TokenList tokensFirst = comparison.getFirstSubmission().getTokenList();
-        TokenList tokensSecond = comparison.getSecondSubmission().getTokenList();
-        Token startTokenFirst = tokensFirst.getToken(match.getStartOfFirst());
-        Token endTokenFirst = tokensFirst.getToken(match.getStartOfFirst() + match.getLength() - 1);
-        Token startTokenSecond = tokensSecond.getToken(match.getStartOfSecond());
-        Token endTokenSecond = tokensSecond.getToken(match.getStartOfSecond() + match.getLength() - 1);
+        List<Token> tokensFirst = comparison.getFirstSubmission().getTokenList();
```

Comment:
maybe we want that getTokenList() returns immutable lists or copies .. just to make the impl clean.

Reply:
Done.